test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -406,8 +409,12 @@ private void triggerDump(OpenJ9DiagnosticsMXBean diagBean) {
 			try {
 				diagBean.triggerDump(agent);
 				if (!(agent.equals("stack"))) {
-					found = findAndDeleteFile(".", files[index]);
-					Assert.assertTrue(found, files[index] + " not found");
+					if ("z/OS".equalsIgnoreCase(os) && "system".equals(agent)) {
```

Comment:
You are calling this 3x in the code, may be useful as a helper method isZOSAgent() or something, especially if going to be used frequently.  (Also, will getSystemProperty("os.name") always return z/OS?  If that return value changes for whatever reason, you only change it in one spot.

Reply:
os.name System property value I think might be using uname -I internally to fetch the OS name. Atleast, the machines I tested on returned z/OS.

Reply:
https://github.com/eclipse/omr/blob/884959f423d0e969ffec7340b695e4ed13daabf3/port/unix/omrsysinfo.c#L520
I expect it will always return "z/OS". Possibly there would be another similar OS in the future, although that seems unlikely as well. Nothing wrong with using common code anyway to avoid potential bugs from having one of the checks incorrect, or when modifications are made in the future.